### ISSUE TITLE
Cannot make rocket fuel

### ISSUE THREAD

**Issue by steinjorn** [Darkosto/SevTech-Ages#1505]
fuel for the rocket takes kerosene and liquid oxygen in the immersive engineering refinery, but to get liquid oxygen, requires mekanism.

* Modpack Version: 3.0.4
* Launcher Used: twitch
* Server/LAN/Single Player: sp
* Resourcepack Enabled?: no
* Optifine Installed?: no

**Comment by daddy77gaming** [comment]
you could should use oxygen from galacticraft i reckon, tho that recipe shows 

![2018-04-06_11 40 54](https://user-images.githubusercontent.com/31905330/38417275-f402dafa-3997-11e8-8383-bdfae3f897ef.png)

which from a previous closed post suggests you should. but seems there is a gating issue in that oxygen collector so i'm also lost at this point

**Comment by daddy77gaming** [comment]
#1059

**Comment by dariumz** [comment]
Yep, it can't be done till age 5, since several of needed items are gated to that age, should be fixed in 3.0.5

**Comment by daddy77gaming** [comment]
guess until an update i'm not going to get there then :)

![2018-04-06_13 03 24](https://user-images.githubusercontent.com/31905330/38420344-af87663c-39a3-11e8-8a1a-473176681509.png)
